recipes/andreasbuhr-cppcoro/all/conanfile.py

```diff
@@ -26,87 +29,92 @@ class AndreasbuhrCppCoroConan(ConanFile):
         "fPIC": True,
     }
 
-    _cmake = None
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
     @property
     def _minimum_compilers_version(self):
         return {
             "Visual Studio": "15",
+            "msvc": "191",
             "gcc": "10",
             "clang": "8",
             "apple-clang": "10",
         }
 
+    @property
+    def _min_cppstd(self):
+        return 17
```

Comment:
This will be an issue for compilers that only have coroutines support for C++20 enabled. This is the case of clang >= 16, where only having `-std=c++20` enables coroutines. With this check, CCI will only try to build for C++ 17. Maybe a solution is to force C++20 as a min required version if the other flag checks have not succeeded?

Reply:
Maybe we could just update this value to be
```return 20 if self.settings.compiler == "clang" and self.settings.compiler.version >= 16 else 17```?

Reply:
There's also a funny edge case that I just ran into: When using libstdc++ with clang, it always required C++20. With libc++, it allows C++17 actually through clang 16, but fails with 17. I'll double check all my tests, but assuming that's correct I'll update the checks.